docs/caching.md

```diff
@@ -0,0 +1,238 @@
+
+---
+title: Caching
+type: docs
+menu: thanos
+---
+
+# Background
+
+Thanos can implement caching layers on multiple components. In general this should speed-up certain processes. There is no definitive guide on how one should setup an entire Thanos stack as each cluster and use-case is different. Especially what KPI's<sup>1</sup>, SLO's<sup>2</sup> and SLI's<sup>3</sup> matter to you, your customers and/or your stack should have influence on the setup.
+
+However it is possible to give guidance on what each component tries to achieve, what it is used for and how you could configure this.
+
+<sub>
+1. service level agreement<br />
```

Comment:
I would love to remove the 'brs' but somehow markdown totally ruins the style for new lines by enters.

Reply:
Weird :thinking: was about to comment about this

Reply:
It adds a 1.5 newline between the 2nd and 3rd line. Not sure how to fix it 🥺